feature/system/management/otg_tests/management_ha_test/management_ha_test.go

```diff
@@ -54,8 +54,24 @@ var (
 		IPv4Len: 32,
 		IPv6Len: 128,
 	}
-
-	mgmtVRF  = "mvrf1"
+	mgmtVRF = map[ondatra.Vendor]string{
+		ondatra.JUNIPER: "mgmt_junos",
+		ondatra.ARISTA:  "mvrf1",
+		ondatra.CISCO:   "mgmtvrf1",
+		ondatra.NOKIA:   "mgmtvrf1",
+	}
+	loopbackIntf = map[ondatra.Vendor]int{
+		ondatra.JUNIPER: 0,
+		ondatra.ARISTA:  1,
+		ondatra.CISCO:   1,
+		ondatra.NOKIA:   1,
+	}
+	loopbackSubIntf = map[ondatra.Vendor]int32{
+		ondatra.JUNIPER: 0,
```

Comment:
As per design I believe Juniper is using subinterface of 100. Please check

Reply:
Updated it in latest commit. Please check.